portal-impl/src/com/liferay/portal/util/PortalImpl.java

```diff
@@ -318,14 +319,14 @@ public PortalImpl() {
 
 		// Computer name
 
-		String computerName = System.getProperty("env.COMPUTERNAME");
+		String computerName = SystemProperties.get("env.COMPUTERNAME");
```

Comment:
@W2jx Your commit message is saying "replace system.properties's property", but here the env.* are not properties we set in system.properties.